test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -9072,6 +9072,49 @@ def forward(self, x):
         x = torch.ones(2, 0, 3)
         self.run_test(M(), (x,))
 
+    def test_broat_cast_tensors(self):
+        class M(torch.nn.Module):
+            def forward(self, x, y):
+                m = torch.broadcast_tensors(x, y)
+                return m
+
+        x = torch.randint(5, (1,))
+        y = torch.randint(5, (5,))
+
+        self.run_test(M(), (x, y))
+
+        x = torch.randint(5, (4, 2, 1, 4))
+        y = torch.randint(5, (2, 3, 1))
+
+        self.run_test(M(), (x, y))
+
+        x = torch.randn(2, 1, 4)
+        y = torch.randn(5, 2, 3, 1)
+
+        self.run_test(M(), (x, y))
+
+    @disableScriptTest()
+    def test_dist_normal(self):
+        class M(torch.nn.Module):
+            def forward(self, x, y):
+                return torch.distributions.Normal(x, y).sample().size(0), x, y
+
+        self.run_test(M(), (torch.tensor([0.0]), torch.tensor([[1.0], [2.0]])))
+        self.run_test(M(), (torch.tensor([0.0]), torch.tensor([1.0])))
+
+        self.run_test(M(), (torch.tensor([[[0.0], [10.0]], [[2.0], [8.0]], [[2.0], [8.0]]]), torch.tensor([[1.0], [3.0]])))
+
+    @disableScriptTest()
+    def test_dist_uniform(self):
```

Comment:
Would it be possible to add a statistical test, and verify the distributions? Maybe have a loop to generate 100 samples and compare mean and variance of samples with expected mean and variance?